### PR TITLE
Fix repeated test runs

### PR DESCRIPTION
**Pull request by SirCmpwn**
If you ran the tests several times, some things would be saved in
localStorage, causing tests to possibly fail.

### CODE REVIEW

File: src/linodes/reducers/index.js [hunk 1]
```diff
@@ -8,16 +8,14 @@ import {
...
-const default_state = {
-  view,
-  selected: { } // set
-};
-
 const array_to_set = arr => arr.reduce((s, v) => ({ ...s, [v]: true }), { });
 
-export function index(state=default_state, action) {
+export function index(state=null, action) {
+  if (state === null) {
```
Comment (eatonphil) [inline src/linodes/reducers/index.js]: Why not no default argument for `state` and just `if (!state) {`? Also, this appears to be identical to what it was before except basically ignoring default arguments. Could you explain how this solved the problem?

Comment (SirCmpwn) [inline src/linodes/reducers/index.js]: `===` > truthy compares imo

Comment (SirCmpwn) [inline src/linodes/reducers/index.js]: And this postpones talking to localStorage until the reducer is invoked.

**Comment by eatonphil** [comment]
ok looks good. ready to merge?

**Comment by SirCmpwn** [comment]
Yes.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**